docs/features/search/search-engines.md

```diff
@@ -237,4 +237,17 @@ search:
     batchSize: 100
 ```
 
-> You can also increase the batch size if you are using a large ES instance.
+### Elasticsearch query config
+
+By default the default settings for the Elasticsearch queries is used. If you need to tweak the fuzziness of the query results you can do this with 2 parameters, fuzziness and prefixLength.
+
+Fuzziness allows you to define the maximum Levenshtein Distance, AUTO is the default and widely accepted standard.
```

Comment:
Can we link out to documentation on what the "Levenshtein Distance" is, I personally have no clue and would read docs if that was included. As for the Vale error in this case your need to add it to this file: https://github.com/backstage/backstage/blob/master/.github/vale/config/vocabularies/Backstage/accept.txt